docs/src/counts.md

```diff
@@ -7,7 +7,7 @@ The package provides functions to count the occurrences of distinct values.
 ```@docs
 counts
 proportions
-addcounts!(r::AbstractArray, x::StatsBase.IntegerArray, levels::StatsBase.IntUnitRange)
+addcounts!(r::AbstractArray, x::StatsBase.AbstractArray{<:Integer}, levels::StatsBase.IntUnitRange)
```

Comment:
```suggestion
addcounts!(r::AbstractArray, x::AbstractArray{<:Integer}, levels::StatsBase.IntUnitRange)
```
I guess `IntUnitRange` could be similarly removed, as it's an alias for `UnitRange{<:Integer}`.